### PR TITLE
[3639140773] Add scripts from core

### PR DESCRIPTION
**Pull request by rYuuk**
## Description

-  Move avatar loader specific scripts from core to avatar loader.

#### Removed

- All data classes specific to AvatarLoader
- Editor settings classes specific to AvatarLoader

## How to Test

-  Best way to test is to check [this](https://github.com/readyplayerme/Unity-SDK/pull/16) PR in Unity-SDK repo 

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.
-   [ ] QA Testing is updated.

## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.

### CODE REVIEW

File: Editor/EditorAssetLoader.cs [hunk 1]
```diff
@@ -22,18 +22,15 @@ public static void CreateSettingsAssets()
             AssetDatabase.Refresh();
             CreateAvatarConfigAssets();
             CreateAvatarLoaderSettings();
-            CreateReadyPlayerMeSettings();
+            CreatePartnerSubDomainSetting();
```
Comment (HarrisonHough) [inline Editor/EditorAssetLoader.cs:25]: SubDomain should be Subdomain. 

Kind of a side note but we could perhaps use PartnerName. or a PartnerSettings object with property Name. However not sure if there would ever be any other "Partner Settings" in future :D 

anyways PartnerSubdomainSettings is fine

**Review by HarrisonHough — CHANGES_REQUESTED** [review]
Looks good, one small thing in the comments (just need to update SubDomain to Subdomain)

**Comment by srcnalt** [comment]
I get error messages for `using ReadyPlayerMe.AvatarLoader;` in Core. Core loads AvatarLoader, and until it is loaded does not have any knowledge of its code base. So we should not mention any code of other modules.

You can import Core into an empty Unity project and see if they show up. 
image

**Comment by HarrisonHough** [comment]
> I get error messages for `using ReadyPlayerMe.AvatarLoader;` in Core. Core loads AvatarLoader, and until it is loaded does not have any knowledge of its code base. So we should not mention any code of other modules.
> 
> You can import Core into an empty Unity project and see if they show up. image

yep this include can be removed, probably just something you missed 

image

**Review by HarrisonHough — APPROVED** [review]
Looks good now

**Pull request merged**